bundles/org.openhab.binding.deconz/src/main/java/org/openhab/binding/deconz/internal/handler/SensorThermostatThingHandler.java

```diff
@@ -181,29 +178,6 @@ protected void valueUpdated(String channelID, SensorState newState, boolean init
 
     @Override
     protected void createTypeSpecificChannels(SensorConfig sensorConfig, SensorState sensorState) {
```

Comment:
Do you need to override this method at all?

Reply:
Yes, it is abstract, needs to be implemented.